AirportPanel/Program.cs

```diff
@@ -0,0 +1,490 @@
+            }
+            Console.BackgroundColor = ConsoleColor.Black;
+            Console.SetCursorPosition(0, 40);
+            Console.WriteLine(s);
+
+            Console.SetCursorPosition(left, top);
+            int number;
+            for (; ; )
+            {
+                string answer = Console.ReadLine();
+                if (int.TryParse(answer, out number))
+                {
+                    if (int.Parse(answer) < 1 || int.Parse(answer) > Enum.GetNames(typeof(CityName)).Length)
+                    {
+                        Console.SetCursorPosition(0, 40);
+                        Console.BackgroundColor = ConsoleColor.Black;
+                        Console.Write("Please enter correctly. " + s);
+                        Console.SetCursorPosition(left, top);
+                    }
+                    else
+                    {
+                        break;
+                    }
+                }
+                else
+                {
+                    Console.SetCursorPosition(0, 40);
+                    Console.BackgroundColor = ConsoleColor.Black;
+                    Console.Write("Please enter correctly. " + s);
+                    Console.SetCursorPosition(left, top);
+                }
+            }
+            return (CityName)number;
+
+        }
+        private static AirlineName InputAirline(int left, int top)
+        {
+            Console.SetCursorPosition(0, 40);
+            Console.Write(new string(' ', 164));
+            string s = "Help list: ";
+            for (int i = 1; i <= Enum.GetNames(typeof(AirlineName)).Length; i++)
+            {
+                s += (AirlineName)i + "-" + i + " ";
+            }
+            Console.BackgroundColor = ConsoleColor.Black;
+            Console.SetCursorPosition(0, 40);
+            Console.WriteLine(s);
+
+            Console.SetCursorPosition(left, top);
+            int number;
+            for (; ; )
+            {
+                string answer = Console.ReadLine();
+                if (int.TryParse(answer, out number))
+                {
+                    if (int.Parse(answer) < 1 || int.Parse(answer) > Enum.GetNames(typeof(AirlineName)).Length)
+                    {
+                        Console.SetCursorPosition(0, 40);
+                        Console.BackgroundColor = ConsoleColor.Black;
+                        Console.Write("Please enter correctly. " + s);
+                        Console.SetCursorPosition(left, top);
+                    }
+                    else
+                    {
+                        break;
+                    }
+                }
+                else
+                {
+                    Console.SetCursorPosition(0, 40);
+                    Console.BackgroundColor = ConsoleColor.Black;
+                    Console.Write("Please enter correctly. " + s);
+                    Console.SetCursorPosition(left, top);
+                }
+            }
+            return (AirlineName)number;
+
```

Comment:
why empty string?)

Reply:
did you find a bug? I always saw the inscription

Reply:
return (AirlineName)number;

}

It is necessary to remove empty lines in the code, just for more convenient code maintenance)